ClosedXML/Excel/XLWorkbook.cs

```diff
@@ -119,7 +119,22 @@ public static XLWorkbook OpenFromTemplate(String path)
 
         public XLEventTracking EventTracking { get; set; }
 
-        #region Nested Type: XLLoadSource
+        /// <summary>
+        /// Counter increasing at workbook data change. Serves to determine if the cell formula
+        /// has to be recalculated.
+        /// </summary>
+        internal long RecalculationCounter { get; private set; }
+
+        /// <summary>
+        /// Notify that workbook data has been changed which means that cached formula values
+        /// need to be re-evaluated.
+        /// </summary>
+        internal void NotifyRecalculationNeeded()
```

Comment:
If I understand this correctly, I think we should rename to `InvalidateFormulas`.